pahfit/component_models.py

```diff
@@ -148,8 +148,8 @@ class S07_attenuation(Fittable1DModel):
     outputs = ('y',)
 
     # Attenuation tau
-    tau_si = Parameter(description="kvt term: amplitude",
-                       default=1.0, min=0.0, max=10)
+    tau_sil = Parameter(description="kvt term: amplitude",
+                        default=1.0, min=0.0, max=10)
 
     @staticmethod
     def kvt(in_x):
```

Comment:
ok, I don't see how I can comment on line 16, but there also weird characters for the "..." in http://www.stsci.edu/~kgordon/pahfit/

Reply:
Can you include a screenshot of what the issue is?  I can't figure out what the problem is - so can't fix it.  :-)

Reply:
In the right column, for BB1D.
![screen shot 2019-03-02 at 9 11 25 am](https://user-images.githubusercontent.com/19981724/53683124-4b3fa280-3ccb-11e9-9f72-cfe6f00eae1f.png)

Reply:
Thanks for the additional info.  Fixed.